src/LibraryManager/Providers/Unpkg/UnpkgCatalog.cs

```diff
@@ -180,14 +180,14 @@ public async Task<CompletionSet> GetLibraryCompletionSetAsync(string libraryName
                     libraryNameStart.StartsWith("@", StringComparison.Ordinal) &&
                     at < 0 || caretPosition < at) 
                 {
-                    IEnumerable<string> packageNames = await NpmPackageSearch.GetPackageNamesAsync(libraryNameStart, CancellationToken.None);
+                    IEnumerable<Tuple<string, string>> packageNamesAndCurrentVersions = await NpmPackageSearch.GetPackageNamesAndCurrentVersionsAsync(libraryNameStart, CancellationToken.None);
 
-                    foreach (string packageName in packageNames)
+                    foreach (Tuple<string, string> nameAndCurrentVersion in packageNamesAndCurrentVersions)
                     {
                         CompletionItem completionItem = new CompletionItem
                         {
-                            DisplayText = packageName,
-                            InsertionText = packageName,
+                            DisplayText = nameAndCurrentVersion.Item1,
```

Comment:
Item1/Item2 syntax suffers from a complete loss of semantic information.  Instead of using them, use tuple deconstruction so you can have names or create a type with members.